src/controller/AbstractDnssdDiscoveryController.cpp

```diff
@@ -22,9 +22,34 @@
 #include <lib/core/CHIPEncoding.h>
 #include <lib/support/logging/CHIPLogging.h>
 
+#include <bitset>
+
 namespace chip {
 namespace Controller {
 
+static bool SameExceptOrder(const chip::Span<const Inet::IPAddress> & source, const chip::Span<const Inet::IPAddress> & destination)
```

Comment:
source/destination names seem off ... those are generally used for 'copy' or equivalent. 

Please use `a,b` or `v1,v2` or anything that makes these symmetrical enough to imply they are not of special usage.

Reply:
I see. Thanks.